docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+      depends_on:
+      - db

```

Comment:
The `depends_on` option is used to specify service dependencies. Note that `depends_on` does not wait for `db` to be 'ready' (i.e., accepting connections), only until it has started. Consider using a wait-for-it script or similar to ensure the database is ready before starting the application.